packages/ek-components/src/components/EkCardBody.vue

```diff
@@ -1,27 +1,49 @@
 <template>
   <div class="card-content">
-    <h5 class="mb-1 title">
-      <VClamp
-        autoresize
-        :maxLines="titleLines"
-      >
-        {{ node.title }}
-      </VClamp>
-    </h5>
-    <!-- FIXME only for debugging: -->
-    <span class="debug-node-id sr-only">{{ node.id }}</span>
-    <div v-if="showChannelIcon" class="align-items-center d-flex mb-3">
-      <EkChannelLogo class="mr-2" :channel="node.channel" size="sm" />
-      <span class="channel-title text-muted text-truncate">{{ node.channel.title }}</span>
-    </div>
-    <p v-else class="mb-1 subtitle text-muted">
-      <VClamp
-        autoresize
-        :maxLines="3"
-      >
-        {{ subtitle }}
-      </VClamp>
-    </p>
+    <v-container>
+
+      <!-- Channel -->
+      <v-row v-if="showChannelIcon">
+        <v-col>
+          <!-- FIXME only for debugging: -->
+          <span class="debug-node-id sr-only">{{ node.id }}</span>
```

Comment:
I was about to suggest "remove this" but I see it was there before.. so I guess nevermind :)